packages/quantic/cypress/e2e/default-2/search-box/search-box.cypress.ts

```diff
@@ -0,0 +1,634 @@
+              });
+
+              scope('when focusing on the search box input', () => {
+                Actions.focusSearchBox(textarea);
+                cy.wait(InterceptAliases.QuerySuggestions);
+
+                Expect.displaySuggestionList(true);
+                Expect.displayClearRecentQueriesButton(false);
+                Expect.numberOfQuerySuggestions(
+                  expectedQuerySuggestions.length
+                );
+                Expect.querySuggestionsEquals(expectedQuerySuggestions);
+              });
+            });
+          });
+
+          describe('when the local storage does not contain recent query suggestions', () => {
+            beforeEach(() => {
+              setRecentQueriesInLocalStorage([]);
+              visitSearchBox({
+                ...defaultOptions,
+                textarea,
+                disableRecentQueries: true,
+              });
+              mockQuerySuggestions(exampleQuerySuggestions);
+            });
+
+            it('should not add the query to the local storage after executing a new search', () => {
+              const expectedQuerySuggestions = exampleQuerySuggestions;
+
+              scope('when loading standalone search box', () => {
+                Expect.displaySearchBoxInput(true, textarea);
+                Expect.displaySearchButton(true);
+              });
+
+              scope('when focusing on the search box input', () => {
+                Actions.focusSearchBox(textarea);
+                cy.wait(InterceptAliases.QuerySuggestions);
+
+                Expect.displaySuggestionList(true);
+                Expect.displayClearRecentQueriesButton(false);
+                Expect.numberOfQuerySuggestions(
+                  expectedQuerySuggestions.length
+                );
+                Expect.querySuggestionsEquals(expectedQuerySuggestions);
+              });
+
+              scope('when selecting a suggestions', () => {
+                const clickedSuggestionIndex = 0;
+                Actions.clickQuerySuggestion(clickedSuggestionIndex);
+
+                Expect.searchWithQuery(
+                  exampleQuerySuggestions[clickedSuggestionIndex],
+                  {
+                    LSkey: recentQueriesLSKey,
+                    queries: [],
+                  }
+                );
+              });
+            });
+          });
+        });
+      });
+
+      describe('accessibility', () => {
+        const exampleQuerySuggestions = ['ABC', 'EFG'];
+
+        describe('aria live', () => {
+          describe('when only query suggestions are displayed', () => {
+            beforeEach(() => {
+              setRecentQueriesInLocalStorage([]);
+              visitSearchBox({...defaultOptions, textarea});
+              mockQuerySuggestions(exampleQuerySuggestions);
+            });
+
+            it('should have the aria live message the properly the number of query suggestions available', () => {
```

Comment:
I think I said before, but this should've been a unit test instead of e2e.
The unit test should test that we're correctly "sending" the message to the aria-live region.
Since we already have unit tests on aria-live that it displays messages it receives, we're covered.

I think it's a bit overkill to test e2e the behavior of both together.
But don't remove it, just keep that in mind for the future.

Reply:
I will also try to address that when adding more unit tests to this components 👍